File: easy/poweroftwo.py
def isPowerOfTwo(n):
    """
    :type n: int
    :rtype: bool
    """
    if n == 1:
        return True
    elif n % 2 == 0:
        for i in range(1, n):
            if 2 ** i == n:
                return True
            elif 2 ** i > n:
                return False
    return False



n = 5010
print(isPowerOfTwo(n))

File: easy/poweroftwo.java
class Solution {
    public boolean isPowerOfTwo(int n) {
        if (n == 1) {
            return true;
        }    
        else if (n % 2 == 0) {
            for (int i = 1; i <= n; i++) {
                int check = (int) Math.pow(2, i);
                if (check == n) {
                    return true;
                }
                else if (check > n) {
                    return false;
                }

            }
        }
        return false;
    }
}